packages/lit-dev-content/site/docs/v2/templates/directives.md

```diff
@@ -1831,7 +1831,10 @@ import {asyncAppend} from 'lit/directives/async-append.js';
 <td class="wide-cell">
 
 ```ts
-asyncAppend(iterable: AsyncIterable)
+asyncAppend(
+  iterable: AsyncIterable,
+  mapper?: (item: Item, index?: number) => unknown
+)
```

Comment:
Sizing now looks good!
One more nit though, I wonder if it should be either
```ts
asyncAppend(
  iterable: AsyncIterable<Item>,
  mapper?: (item: Item, index?: number) => unknown
)
```
or replace `Item` with `T`, or make it `item: unknown` as well?